src/cartridge/src/cartridge.cpp

```diff
@@ -14,21 +14,27 @@
 namespace cartridge {
 
 Cartridge::Cartridge(std::string rom_path) {
-	// Open the file
-	auto rom_file =
-	    std::ifstream(rom_path, std::ifstream::in | std::ios::binary);
-	auto result = std::vector<uint8_t>();
-
-	// Compute file size
-	rom_file.seekg(0, rom_file.end);
-	auto rom_file_size = rom_file.tellg();
-	rom_file.seekg(0);
-
-	// Read the file
-	auto byte_input = std::vector<char>(rom_file_size, '\0');
-	rom_file.read(&byte_input[0], rom_file_size);
-
-	data = std::vector<uint8_t>(byte_input.begin(), byte_input.end());
+	try {
```

Comment:
It looks like you've accidentally pushed your older commits to this branch. You can fix this by _rebasing_ your current branch against master, and then force pushing again to this branch. Be careful when force pushing!